src/rover/main.cpp

```diff
@@ -0,0 +1,174 @@
+//#define DEBUG
+#define TIME
+
+#include "camera.hpp"
+#include "../network/network.hpp"
+
+#include <cstddef>
+#include <iostream>
+#include <vector>
+#include <cstring>
+#include <cstdlib>
+#ifdef TIME
+	#include <chrono> // Time keeping from stdlib
+	using namespace std::chrono;
+#endif
+
+
+const unsigned int CAMERA_MESSAGE_SIZE = 65000;
+
+int main() {
+	/*
+	 * INITILIZATION: Rover Objects
+	 * OPTIMIZE: Consider making a rover class with an iteratable list of cameras
+	 */
+	unsigned int frame_counter = 0;
+	// Open all cameras
+	camera::CaptureSession session0;
+	camera::CaptureSession session1;
+	camera::CaptureSession session2;
+	camera::CaptureSession session3;
+	camera::Error err0 = camera::open(&session0, "/dev/video0", 1280, 720);
```

Comment:
Those numbers should be `#define'd` or `const` at a header file

Reply:
Done!